reference-manual/src/main/asciidoc/expert-zone/dag.adoc

```diff
@@ -263,7 +263,7 @@ is often significantly faster and it's safe to use on a local edge.
 On a distributed edge it is not a safe strategy in general because
 `hashCode()` 's contract does not require repeatable results across
 JVMs or even different instances of the same JVM version. If a given
-class's Javadoc explicitly specifies the hashing function it uses, then
+class' Javadoc explicitly specifies the hashing function it uses, then
```

Comment:
It should actually stay as "class's". Technically both are acceptable, no point in changing from one to the other and frankly `class'` doesn't looks so familiar to me.

Reply:
Actually I wanted it to be consistent with other spelling `members'`, `items'`...

Reply:
"members" is plural, "class" is singular. See https://english.stackexchange.com/q/1073/232121